File: Revisao/classificacao_estado_civil.py
# Programa para identificar estado civil
escolha = input("")

# Pegamos apenas a primeira letra e colocamos em minúscula
letra = escolha[0].lower()

if letra == 'c':
    print("Casado")
elif letra == 's':
    print("Solteiro")
elif letra == 'd':
    print("Divorciado")
else:
    print("Alguma letra errada")

File: Revisao/classificacao_estado_civil.c
// Programa para identificar estado civil
#include <stdio.h>
int main (){
    char escolha[12];

    scanf ("%s", &escolha);

    switch (escolha[0])
    {
    case 'c':
        printf("Casado");
        break;
    
    case 's':
        printf("Solteiro");

    case 'd':
        printf("Divorciado");
    
    default:
        printf("Alguma letra errado")
        break;
    }


    return 0;
}